docs/Makefile

```diff
@@ -1,5 +1,9 @@
 # Minimal makefile for Sphinx documentation
 #
+# copy current tutorials
+copy:
+	cp -r ../tutorials/* source/tutorials/
+
```

Comment:
Alternatively, we could place the tutorials in that directory. It's likely someone would access the tutorials through the binder or the documentation anyway. Not required though